misc/org.projecthamster.hamster-gtk.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=hamster-gtk
```

Comment:
IMHO "frontend client"  is all the user cares about. Instead of "the image opener" we call distinct packages different names even if they do the same and may even use the same libs for functionality. As there are already ppl working on a QT version it is easy to see that one repo may offer multiple packages at the same time that are "hamster time trackers" in their own right. The name should reflect that they are but also how they differ I think. As I said: I am happy to discuss this once it is more relevant, right now I think it would be nice to wrap things up :)